ldes-server-port-fetch-rest/pom.xml

```diff
@@ -52,6 +52,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+
+        <!-- LOGGING -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-aop</artifactId>
+        </dependency>
```

Comment:
We should normally only need the starter in the application. In here, we probably need the underlying libraries, no ?

Reply:
Yeah sorry, this was just me struggling a little with the dependencies and module-info.java. I will delete the dependency